bot/cogs/source_code_cog.py

```diff
@@ -180,8 +180,12 @@ def chunk_iterable(self, iterable, chunk_size):
             yield iterable[i:i + chunk_size]
     
     def process_source(self, source: str, line_start: int = None, line_stop: int = None):
-        split_source = [f'{i:03d} |  {value}' for i, value in enumerate(source.splitlines())]
-        filtered_source = split_source[line_start or 0: line_stop or len(source)]
+        split_source = [f'{i+1:03d} |  {value}' for i, value in enumerate(source.splitlines())]
+
+        if line_start is not None and line_start <= 0:
```

Comment:
this can be 

if line_start and line_start <= 0:
	...

this is called short circuiting

Reply:
check the commit after that